apps/design/frontend/src/geography_screen.tsx

```diff
@@ -538,66 +545,116 @@ function PrecinctForm({
-                        value: district.id,
-                        label: district.name,
-                      }))}
-                      value={[...split.districtIds]}
-                      onChange={(districtIds) =>
-                        setPrecinct({
-                          ...precinct,
-                          splits: replaceAtIndex(precinct.splits, index, {
-                            ...split,
-                            districtIds: districtIds as DistrictId[],
-                          }),
-                        })
-                      }
-                    />
-                    <InputGroup label="Election Title Override">
-                      <input
-                        type="text"
-                        value={split.nhCustomContent.electionTitle ?? ''}
-                        onChange={(e) =>
-                          setPrecinct({
-                            ...precinct,
-                            splits: replaceAtIndex(precinct.splits, index, {
+                      <InputGroup label="Election Title Override">
+                        <input
+                          type="text"
+                          value={split.nhCustomContent.electionTitle ?? ''}
+                          onChange={(e) =>
+                            setSplit(index, {
                               ...split,
                               nhCustomContent: {
                                 ...split.nhCustomContent,
                                 electionTitle: e.target.value,
                               },
-                            }),
-                          })
-                        }
-                      />
-                    </InputGroup>
-
-                    <Button onPress={() => onRemoveSplitPress(index)}>
-                      Remove Split
-                    </Button>
-                  </Column>
-                </Card>
-              ))}
+                            })
+                          }
+                        />
+                      </InputGroup>
+
+                      <div>
+                        <FieldName>Clerk Signature Image</FieldName>
+                        {split.nhCustomContent.clerkSignatureImage && (
+                          <img
+                            src={`data:image/svg+xml;base64,${Buffer.from(
+                              split.nhCustomContent.clerkSignatureImage
+                            ).toString('base64')}`}
+                            alt="Clerk Signature"
+                            style={{ height: '3rem', marginBottom: '0.5rem' }}
+                          />
+                        )}
+                        <FileInputButton
+                          accept="image/svg+xml"
+                          onChange={(e) => {
+                            const file = e.target.files?.[0];
+                            if (!file) return;
+                            const reader = new FileReader();
```

Comment:
I suggest, before reading this file, that we pick a maximum file size. Perhaps 5MB? `File` has a `size` property in bytes that we can check.

Reply:
Good idea 7fa4595c5